artifacts/config_files/sfall-mods.ini

```diff
@@ -4,8 +4,14 @@
 ;  42 - SHIFT key
 Key=42
 
-; Set to 1 also highlight containers
-Containers=1
+; Set to 1 to also highlight containers
+Containers=0
```

Comment:
Why not enable by default? Users will still have to install these mods again when they get sfall 4.0, so we don't have to be constrained to previous defaults.